rstudio/README.MD

```diff
@@ -0,0 +1,21 @@
+# RStudio Server
+
+This initialization action installs the latest version of [RStudio Server](https://www.rstudio.com/products/rstudio/#Server) on a master node within a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster.
+
+## Using this initialization action
+You can use this initialization action to create a new Dataproc cluster with RStudio Server installed by:
+
+1. Uploading a copy of this initialization action (`rstudio.sh`) to [Google Cloud Storage](https://cloud.google.com/storage).
```

Comment:
Heh I know a lot of init action README's have instructions to copy a script to your own GCS bucket. I'd rather have all of them avoid that and just have a `clusters create` command that uses our mirror of this repo on GCS:

```
gcloud dataproc clusters create <CLUSTER_NAME> \
  --initialization-actions gs://dataproc-initialization-actions/rstudio/rstudio.sh
```

Reply:
I will update README after the script updates are done.

Reply:
Updated